### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by DariaFesiun**
- [DEMO LINK](https://DariaFesiun.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://DariaFesiun.github.io/layout_search-bar-airbnb/report/html_report/)

- [x] Icon implemented using background-image CSS property
- [x] Inputs are written inside of 'form' tag with correctly passed attributes
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,89 @@
...
+
+.form[data-qa='big'] {
+  padding-left: 62px;
+  height: 70px;
+}
+
+.form[data-qa='small'] {
+  padding-left: 33px;
+  height: 42px;
+}
```
Comment (IvanFesenko) [inline src/style.css:30]: Add modifier class and use, those selector only for test proposes

File: src/style.css [hunk 2]
```diff
@@ -1 +1,89 @@
...
+
+.form_input--small {
+  font-size: 14px;
+  height: 20px;
+}
+
+.form_input--big::placeholder,
+.form_input--small::placeholder {
+  font-family: Avenir, sans-serif;
```
Comment (IvanFesenko) [inline src/style.css:54]:
```suggestion
  font-family: inherit;
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Please provide demo link and report link

- Content not centered
2024-04-09_12h25_40
2024-04-09_12h26_40

File: src/index.html [hunk 3]
```diff
@@ -10,23 +10,37 @@
...
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="form"
+      data-qa="big"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:23]: You need to use the form tag instead of div

File: src/index.html [hunk 4]
```diff
@@ -10,23 +10,37 @@
...
+      <input
+        class="form_input form_input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+      <div class="icon icon--big"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:30]: Redundant, add icon for the input as abackground
```suggestion
```

File: src/index.html [hunk 5]
```diff
@@ -10,23 +10,37 @@
...
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="form"
+      data-qa="small"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:36]: The same as above

File: src/index.html [hunk 6]
```diff
@@ -10,23 +10,37 @@
...
+      <input
+        class="form_input form_input--small"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+      />
+      <div class="icon icon--small"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html:43]:
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
You need to fix the demo link
image

File: readme.md [hunk 7]
```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://DariaFesiun.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://DariaFesiun.github.io/layout_search-bar-airbnb/report/html_report/)
```
Comment (volodymyr-soltys97) [inline readme.md:29]: Add these links to the PR description
image

File: src/index.html [hunk 8]
```diff
@@ -10,23 +10,45 @@
...
+        class="form_input form_input--big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+      <img
+        class="icon icon--big"
+        src="/src/images/Search.svg"
+        alt="Icon big"
+      />
```
Comment (volodymyr-soltys97) [inline src/index.html:34]: Redundant, because you need to add icon as background
```suggestion
```

File: src/index.html [hunk 9]
```diff
@@ -10,23 +10,45 @@
...
+        class="form_input form_input--small"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+      />
+      <img
+        class="icon icon--small"
+        src="/src/images/Search.svg"
+        alt="Icon small"
+      />
```
Comment (volodymyr-soltys97) [inline src/index.html:51]: The same as above
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, you need to fix the demo link, try to make a deploy again, if you need a help feel free in the fe_chat
image

**Pull request closed without merging**